data/db-room/src/main/java/io/github/droidkaigi/confsched2019/data/db/entity/mapper/ContributorDataMapperExt.kt

```diff
@@ -0,0 +1,18 @@
+package io.github.droidkaigi.confsched2019.data.db.entity.mapper
+
+import io.github.droidkaigi.confsched2019.data.db.entity.ContributorEntityImpl
+import io.github.droidkaigi.confsched2019.data.api.response.ContributorItemResponse
+
+fun List<ContributorItemResponse>.toContributorEntityImpl(): List<ContributorEntityImpl> = map {
+    it.toContributorEntityImpl()
+}
+
+fun ContributorItemResponse.toContributorEntityImpl(): ContributorEntityImpl{
+    return ContributorEntityImpl(
+        id = requireNotNull(id),
```

Comment:
🤔 in the `ContributorItemResponse`, `id` and `name` are listed as being Nullable. maybe its better to make them non-null in the response instead, that way we don't need to use `requireNotNull` here 👍

Reply:
Thanks, I fixed it.

Reply:
Sorry, @panpanini says that please make Response filed to non-null.
But your changes make entry field nullable type.
Can you confirm it?